src/panel/poi_bloc/images_panel.js

```diff
@@ -2,9 +2,20 @@ import ImagesPanelView from '../../views/poi_bloc/images.dot'
 import Panel from "../../libs/panel";
 
 export default class ImagesPanel {
-    constructor(block){
-      this.panel = new Panel(this, ImagesPanelView)
-      this.name = block.name
-      this.block = block
+  constructor() {
```

Comment:
this constructor receives all necessary details to render the block : 
```suggestion
  constructor(block, poi, options) {
```